playground/fastify/package.json

```diff
@@ -6,21 +6,20 @@
     "start": "ts-node ./src/server.ts"
   },
   "dependencies": {
-    "@clerk/backend": "file:.yalc/@clerk/backend",
-    "@clerk/fastify": "file:.yalc/@clerk/fastify",
-    "@clerk/shared": "file:.yalc/@clerk/shared",
-    "@clerk/types": "file:.yalc/@clerk/types",
```

Comment:
should we retain yalc usage here for the playground?

Reply:
I removed it because when you use `clerk-dev-cli` or `secco` it gets in the way. If we want to dogfood the dev CLI we need to remove yalc ;)